lib/src/test/java/io/ably/lib/test/realtime/RealtimeChannelTest.java

```diff
@@ -21,6 +21,7 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
+import javax.management.DescriptorAccess;
```

Comment:
Hm, I have no idea what is it doing here and why is it needed here, will remove it